src/main/java/uk/ac/cam/cl/dtg/segue/dos/users/UserExternalAccountChanges.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 James Sharkey
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.segue.dos.users;
+
+/**
+ *  A class to represent the state of a user that needs synchronising with an external provider.
+ */
```

Comment:
Just for clarity, is this used strictly as our internal DO or does it get serialised and traverse the network to Mailjet - don't mind if it does but could be worth putting in the javadoc for clarity?

Reply:
MailJet doesn't allow you to create a user and update their properties at the same time, and you cannot pass the JSON directly but JSON describing the JSON of the user; so this is entirely internal and split into pieces to be used in the various cases. I will update the JavaDoc.